config/QUnit.config.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: default
+categories: [config]
+title: QUnit.config
+description: General configuration for QUnit. Check the description of each option for details.
+---
+
+## `QUnit.config`
+
+Configuration for QUnit. QUnit has a bunch of internal configuration defaults, some of which are useful to override. Check the description for each option for details.
+
+### `QUnit.config.altertitle` (boolean) | default: `true`
+
+By default, QUnit updates document.title to add a checkmark or x-mark to indicate if a testsuite passed or failed. This makes it easy to see a suites result even without looking at a tab's content.
+
+If you're dealing with code that tests `document.title` changes or have some other problem with this feature, set this option to false to disable it.
+
+### `QUnit.config.autostart` (boolean) | default: `true`
+
+By default, QUnit runs tests when `load` event is triggered on the `window`. If you're loading tests asynchronously, you can set this property to `false`, then call `QUnit.start()` once everything is loaded. See below for an example.
+
+### `QUnit.config.collapse` (boolean) | default: `true`
+
+By default, QUnit's HTML reporter collapses consecutive failing tests showing only the details for the first failed test. The other tests can be expanded manually with a single click on the test title. Setting this value to `false` will expand the details for every failing test.
+
+### `QUnit.config.current` (object)
+
+This object isn't actually a configuration property, but is listed here anyway, as its exported through `QUnit.config`. This gives you access to some QUnit internals at runtime. See below for an example.
```

Comment:
s/as its exported/as it's exported

I assume this is a long-standing typo rather than anything introduced in this PR, but all the same...